test/functional/CacheManagement/src/tests/sharedclasses/TestUtils.java

```diff
@@ -1509,18 +1507,14 @@ public static boolean removeSharedMemoryForCache(String cacheName) {
 		return RunCommand.lastCommandStderrLines;
 	}
 	
-	public static boolean isIBM() {
-		return System.getProperty("java.vm.vendor").toLowerCase().contains("ibm");
-	}
-	
 	public static boolean isOpenJ9() {
 		return System.getProperty("java.vm.vendor").toLowerCase().contains("openj9");
 	}
 	
 	public static String removeJavaSharedResourcesDir(String dir) {
-		if ( dir.contains("javasharedresources") ) {
+		if ( dir.endsWith(java.io.File.separator + "javasharedresources") ) {
```

Comment:
It'll be more robust to handle directories end with `java.io.File.separator + javasharedresources + java.io.File.separator` as well. You have extra spaces before `dir` and `) {`

Reply:
The pass in parameter is the return value of getCacheDir(string, boolean), which do not end with java.io.File.separator.

Reply:
> return value of getCacheDir(string, boolean), which do not end with java.io.File.separator

Might change in the future.

Reply:
Updated.